backend/src/main/java/com/github/NicolaiKopka/web_page_services/user_pages/user_favorites/UserFavoritesService.java

```diff
@@ -104,4 +104,10 @@ public UserFavoritesSaveObject removeUserPlaylist(String username, String playli
         favoritesObject.removeUserPlaylist(playlistName);
         return userFavoritesRepo.save(favoritesObject);
     }
+
```

Comment:
if you always need the user id for db requests you could maybe save it instead of username in jwt token to save one request